projects/plugins/jetpack/class.jetpack-post-images.php

```diff
@@ -897,7 +897,7 @@ public static function fit_image_url( $src, $width, $height ) {
 		}
 
 		// If WPCOM hosted image use native transformations.
-		$img_host = wp_parse_url( $src, PHP_URL_HOST );
+		$img_host = wp_parse_url( $src, PHP_URL_HOST ) ?? '';
 		if ( str_ends_with( $img_host, '.files.wordpress.com' ) ) {
```

Comment:
This works for the case that the URL can be parsed but has no host part. It won't resolve the case if `$src` is not parseable (i.e. when `wp_parse_url()` returns false). Is that worth caring about?
```suggestion
		$img_host = wp_parse_url( $src, PHP_URL_HOST );
		if ( $img_host && str_ends_with( $img_host, '.files.wordpress.com' ) ) {
```

Reply:
I did read the docs, but incorrectly. 😓 Your suggestion covers both scenarios.